doc/Learn/Tutorials/Authentication/HowTo-Cookies.md

```diff
@@ -0,0 +1,13 @@
+---
+uid: Learn.Tutorials.Authentication.HowToCookieAuthorization
+---
+# How-To: Using Cookies to Authorize
+
+`Uno.Extensions.Authentication` provides you with a consistent way to add authentication to your application. It is recommended to one of the built in `IAuthenticationService` implementations. This tutorial will use the custom authorization to validate user credentials.
```

Comment:
It is recommended to do what with one of your build? I do not understand that sentence.